apps/prairielearn/src/question-servers/freeform.ts

```diff
@@ -1281,7 +1291,7 @@ export async function render(
             }
             for (const type in extensionDynamic) {
               for (const key in extensionDynamic[type]) {
-                if (!Object.hasOwn(dynamicDependencies[type], key)) {
+                if (!(key in dynamicDependencies[type])) {
```

Comment:
Nit, why this change?

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/12620/commits/5671c6a44b2c579d0bce9c61b26863eea521035f

Reply:
Backed out in b6045cd.